packages/xo-web/src/xo-app/logs/log-alert-header.js

```diff
@@ -9,7 +10,11 @@ import ReportBugButton, { CAN_REPORT_BUG } from 'report-bug-button'
 import Tooltip from 'tooltip'
 import { get } from 'xo-defined'
 import { injectState, provideState } from '@julien-f/freactal'
-import { subscribeBackupNgLogs } from 'xo'
+import { isEmpty } from 'lodash'
+import { runBackupNgJob, subscribeBackupNgLogs } from 'xo'
+
+const isFailureTask = ({ status }) =>
+  status !== 'success' && status !== 'pending'
```

Comment:
This does not follow the requirements in the issue.

Reply:
We need also to be able to restart the interrupted backup jobs.

Reply:
My bad it's correct, I mixed up *pending* with *skipped*.